lms/djangoapps/static_template_view/urls.py

```diff
@@ -3,34 +3,34 @@
 """
 
 from django.conf import settings
-from django.conf.urls import patterns, url
+from django.conf.urls import url
 
-urlpatterns = (
-    'static_template_view.views',
+from static_template_view import views
 
-    # TODO: Is this used anymore? What is STATIC_GRAB?
```

Comment:
It looks like this comment has been puzzling for more than 5 years: https://github.com/edx/edx-platform/commit/186e5fc5e1becc155bb136026ac3aa90ec18e2db#diff-ffae1272e4176b71b1839d69f055084bR25

Reply:
To satisfy curiosity on this I am going to push a change with it removed to see what happens.